File: PublishText.js
import AWS from 'aws-sdk';
AWS.config.update({region: 'us-east-1'});
function snsRequest(mensagem){
    var params = {
        Message: mensagem,
        TopicArn: 'arn:aws:sns:us-east-1:123456789012:myTopic'
    };
    var publishTextPromise = new AWS.SNS({apiVersion: '2010-03-31'}).publish(params).promise();
  }
exports.handler = (event, context, callback) => {
  snsRequest("Enviando mensagem")
};

File: PublishText.py
import boto3

def send_sns(message, subject):
    client = boto3.client("sns")
    topic_arn = 'arn:aws:sns:us-east-1:085959936702:snstopic'
    client.publish(
        TopicArn=topic_arn, Message=message, Subject=subject)

def lambda_handler(event, context):
    message = "Hello from lambda!"
    subject = "From  Lambda"
    send_sns(message, subject)